tensorflow/lite/experimental/micro/kernels/all_ops_resolver.cc

```diff
@@ -29,13 +29,19 @@ TfLiteRegistration* Micro_Register_FULLY_CONNECTED() {
 TfLiteRegistration* Register_SOFTMAX();
 TfLiteRegistration* Micro_Register_SOFTMAX() { return Register_SOFTMAX(); }
 
+TfLiteRegistration* Register_CONVOLUTION_REF();
+TfLiteRegistration* Micro_Register_CONVOLUTION_REF() {
+  return Register_CONVOLUTION_REF();
+}
+
 AllOpsResolver::AllOpsResolver() {
   AddBuiltin(BuiltinOperator_DEPTHWISE_CONV_2D,
              Micro_Register_DEPTHWISE_CONV_2D());
   AddBuiltin(BuiltinOperator_FULLY_CONNECTED, Micro_Register_FULLY_CONNECTED(),
              /* min_version */ 1,
              /* max_version */ 2);
   AddBuiltin(BuiltinOperator_SOFTMAX, Micro_Register_SOFTMAX());
+  AddBuiltin(BuiltinOperator_CONV_2D, Micro_Register_CONVOLUTION_REF());
```

Comment:
I would prefer to make it consistent. Probably stick with CONV_2D
e.g.
AddBuiltin(BuiltinOperator_CONV_2D, Micro_Register_CONV_2D());